app/javascript/packs/OAS3Autocomplete.js

```diff
@@ -32,41 +34,40 @@ const addAutocompleteToParam = (param: Param, accountData: AccountData): Param =
     : param
 }
 
-const getPathParameters = (path) => (
-  Object.keys(path).reduce((params, item) => (
-    {
-      ...path[item],
-      [item]: item === 'parameters'
-        ? path.parameters
-        : path[item].parameters
-    }
-  ), {})
-)
-
-const injectParametersField = (key, value, accountData) => {
-  const parametersArray = value.map(param => X_DATA_ATTRIBUTE in param ? addAutocompleteToParam(param, accountData) : param)
-  return (key === 'parameters') ? parametersArray : { ...value, parameters: parametersArray }
+const injectParametersToPathOperation = (pathOperation: PathOperationObject, accountData: AccountData): PathOperationObject => {
+  const operationParameters = pathOperation.parameters || undefined
```

Comment:
I think the second undefined is redundant:
```suggestion
  const operationParameters = pathOperation.parameters
```